test/Performance/DynamoPerformanceTests/packages.config

```diff
@@ -51,7 +51,6 @@
   <package id="System.Threading.Tasks.Parallel" version="4.3.0" targetFramework="net461" />
   <package id="System.Threading.Thread" version="4.3.0" targetFramework="net461" />
   <package id="System.ValueTuple" version="4.5.0" targetFramework="net461" requireReinstallation="true" />
-  <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net461" />
```

Comment:
Self-serve CI was complaining about conflicting versions of `System.IO` and `System.Xml.ReaderWriter`.